src/python/setup.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+
+def readme():
+    with open("README.md") as desc:
+        return desc.read()
+
+
+setup(
+    name="hirte",
+    version="1.0.0",
+    description="Python wrapper implementation for Hirte.",
+    long_description=readme(),
+    author="Hirte developers",
+    url="https://github.com/containers/hirte/",
+    license="GPLv2",
+    install_requires=[
+        "dasbus",
+    ],
+    package_dir={"": "."},
+    packages=find_packages("."),
+    include_package_data=True,
+    package_data={"hirte": ["py.typed"]},
+    zip_safe=True,
+    keywords="hirte python",
+    classifiers=[
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
We could add `License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+) ` to this list

Reply:
Please use SPDX license identifiers, so `GPL-2.0-or-later` in our case:

https://spdx.org/licenses/

Reply:
Sorry, I meant to use SPDX within the license string on line 17